tests/test_bindings.py

```diff
@@ -394,6 +394,46 @@ def test_named_detail_action(self):
 
         self.assertEqual(json_content['payload'], expected)
 
+    def test_bad_permission_reply(self):
+        mock_has_perm = Mock()
+        mock_has_perm.return_value = False
+        with patch.object(TestModelResourceBinding, 'has_permission', mock_has_perm):
+            json_content = self._send_and_consume('websocket.receive', self._build_message('testmodel',{
+                'action': 'named_detail',
+                'pk': 546,
+                'data': {},
+                'request_id': 'client-request-id'
+            }))
+
+            expected = {
+                'action': 'named_detail',
+                'errors': ['Permission Denied'],
+                'data': None,
+                'response_status': 401,
+                'request_id': 'client-request-id'
+            }
+
+            self.assertEqual(json_content['payload'], expected)
+            mock_has_perm.assert_called()
```

Comment:
I think the `assert_called` method is incorrect, or at least I couldn't find it in the [docs](https://docs.python.org/3.5/library/unittest.mock.html).

I think this would work instead
```python
self.assertEqual(mock_has_perm.called, True)
```